<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>enter your name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_response">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Show your name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonClickMe">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Click</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonClickMe</sender>
   <signal>clicked()</signal>
   <receiver>label_response</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
